playground/src/importer/importer.js

```diff
@@ -71,7 +72,7 @@ class URLImport {
     );
   }
 
-  onSubmit(e) {
+  onSubmitURL(e) {
```

Comment:
Style nit: following most [style guides](https://developer.android.com/kotlin/style-guide#camel_case) this would be `onSubmitUrl`